runtime/builtin_types_test.go

```diff
@@ -422,3 +422,24 @@ func TestBuiltinSetAttr(t *testing.T) {
 		}
 	}
 }
+
+// TODO: Improvment unit tests codes into catch user's input and output.
```

Comment:
I'd want to see some additional tests that exercise the whole builtinRawInput() function before I approve this PR. It's important that new code is fully tested because the probability that somebody retroactively adds tests is low.